ballerina/tests/resources/expected_results/multiple_operations_with_duplicate_variables.json

```diff
@@ -0,0 +1,13 @@
+{
+    "errors": [
+        {
+            "message": "Variable \"$profileId\" got invalid value 1;Expected type String. String cannot represent value: 1",
```

Comment:
This does not express the presence of a duplicate variable. It's just a type mismatch. We have to check the error when a duplicate variable is defined with the same type, I think.